README.md

```diff
@@ -3,79 +3,105 @@ Cancer Systems Biology, Section of Bioinformatics, Department of Health and Tech
-##### install:
-conda create --prefix ./env_Basal -c conda-forge r-base=4.1.3 r-devtools
-##### to activate
-conda activate ./env_Basal
+All the analyses have been performed on a GNU/Linux server.
 
-## General information
+In order to reproduce the paper data, you will need to set up a conda environment
+on which the expected version of R and the required packages will be installed;
+this requires being able to run Anaconda by means of the `conda` executable.
 
-The scripts in /scripts/ and data in /data/raw_data/ is the only things needed to run this entire projct.
+If you don't have access to `conda` please see the [Miniconda installer page](https://docs.conda.io/en/latest/miniconda.html) on instructions on how to install Miniconda.
 
-To run entire project through terminal type (requires R): 
-$ ./Run_Basal.sh               
+Once you have access to `conda`, you can
 
-Otherwise it is possible to run the casestudy though Rstudio by running ./scripts/00_init.R
+1. clone our github repository into a local directory on your local machine:
 
-## FOLDERS:
+```
+git clone https://github.com/ELELAB/Moonlight2_DMA_basal_like.git
+cd Moonlight2_DMA_basal_like
+```
 
-### /data/
-The data generated by moonlight:
-Basal_FEA.rda, Basal_URA.rda, Basal_GRN.rda, Basal_PRA.rda, Basal_DMA.rda
-This data is stored at https://osf.io/eq9wj/
+2. create a virtual environment using conda activate it:
 
-#### /rawdata/
-The data that is not generated in this project, but was used as input to generate all other data and results in this project. 
-* Mutation from TCGA-BRCA "mutations.csv"
-* Differential Gene expression from TCGA-BRCA "BRCA_BRCA.Basal_dataDEGs.rda"
-* Normal counts of gene expression "BRCA_BRCA.Basal_dataFilt_HUGO.rda"
-* Driver genes predicted by GUST "GUST_BRCA.csv"
-* List of transcription factors from TRRUST "trrust_rawdata_human.tsv"
-* List of human kinases from Kinhub "kinhub.tsv"
-* List of PAM50 genes "pam50_list.txt"
-This data is stored at https://osf.io/eq9wj/
+```
+conda create --prefix ./env_Basal -c conda-forge r-base=4.2 r-pacman r-curl r-ragg
+conda activate ./env_Basal
+```
 
-### /scripts/
-The script 00_init.R loads and checks the prirequirements to run this project.
-All scripts are then run in the correct order.
-The script are numbered and the results from each are numbered accordingly.
+3. run the analyis:
 
-### /results/
-The three additional tables of data from DMA.This folder containts plots and tables used in the paper.The numbers refere to the script they are generated from.
+```
+$ bash ./Run_Basal.sh
+```
 
-### PAPER CONTENTS 
+This script will:
 
+1. Install in the environment all the necessary R packages, including Moonlight2R
+from [our GitHub repository](https://www.github.com/ELELAB/Moonlight2R)
+2. Download from [our OSF repository](https://osf.io/eq9wj/) the `data` folder
+that contains data required for the analysis (see below). This takes approximately
+700MB of disk space.
+3. perform the analysis
```

Comment:
These changes are updated in the Run_Basal.sh script, right? But I don't see that this updated bash script has been added - am I missing something?

Reply:
no, these are included in the `00_init.R` script